instaloader/instaloader.py

```diff
@@ -1418,6 +1444,7 @@ def download_profiles(self, profiles: Set[Profile],
         :param profile_pic: not :option:`--no-profile-pic`.
         :param posts: not :option:`--no-posts`.
         :param tagged: :option:`--tagged`.
+        :param reels: :option:`--reels`.
```

Comment:
It would be nice to also have a `.. versionchanged:: 4.14` directive for the new parameter.